mbrl/diagnostics/training_browser.py

```diff
@@ -172,12 +173,13 @@ def onExperimentsSelectionChanged(self, sel1=None, sel2=None):
                 mean_series = df.mean(axis=1)
                 var_series = df.std(axis=1)
                 
-                mean_line = self.axes.plot(time_series, mean_series)
+                mean_line = self.axes.plot(time_series, mean_series, label=self.experiment_results[rowIndex])
```

Comment:
Is there any way to have a button that has the option to save the plot to some folder? Even something as simple as a showing a text box for the user to write the path and use `matplotlib` to save it somewhere.